src/platform_impl/ios/window.rs

```diff
@@ -353,8 +356,10 @@ impl Window {
                 Some(Fullscreen::Exclusive(ref video_mode)) => {
                     video_mode.video_mode.monitor.ui_screen() as id
                 }
-                Some(Fullscreen::Borderless(ref monitor)) => monitor.ui_screen() as id,
-                None => monitor::main_uiscreen().ui_screen(),
+                Some(Fullscreen::Borderless(Some(ref monitor))) => monitor.inner.ui_screen(),
+                Some(Fullscreen::Borderless(None)) | None => {
+                    monitor::main_uiscreen().ui_screen() as id
+                }
```

Comment:
I'd assume this is because we don't have any monitor yet? Should this be documented maybe that it's not something supported at startup?

Reply:
It's basically that we're asking for a current monitor. However if you look at ios it doesn't really have a fullscreen in a way you'd know. It just always sets screen and it's always fullscreened, that branch just picks the 'preffered monitor', which is the current one here. Not sure whether such thing should be documented, since it's there for a long time.